### PR TITLE
Update node opcua

### PR DESCRIPTION
**Pull request by erossignon**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,6 +25,7 @@
         "lint": "npm run lint --silent --workspaces --if-present",
         "lint:fix": "npm run lint:fix --silent --workspaces --if-present",
         "format": "prettier --write . && npm run format --silent --workspaces --if-present",
+        "update:opcua": "npx -y npm-check-updates -u --deep -f \"node-opcua*\" && npm install --workspaces",
```
Comment (danielpeintner) [inline package.json:28]: Mhh, needed/wanted ?

Comment (erossignon) [reply package.json:28]: Yes, in the new binding there will be a dozen of node-opcua-* modules to update, this script will become handy.

Comment (danielpeintner) [reply package.json:28]: I see  (https://github.com/eclipse/thingweb.node-wot/pull/663#pullrequestreview-874492744)

**Review by danielpeintner — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (relu91) [reply package.json:28]: Can't this be moved to the `binding-opcua` `package.json`?

Comment (erossignon) [reply package.json:28]: I'll drop this PR , I don't thnk it is needed anymore as the new binding will overwriteit.

**Pull request closed without merging**